lib/model/model.go

```diff
@@ -425,7 +425,13 @@ func (m *model) stopFolder(cfg config.FolderConfiguration, err error) {
 	m.fmut.RUnlock()
 
 	if ok {
-		m.RemoveAndWait(token, 0)
+		if err := m.RemoveAndWait(token, 0); err != nil {
```

Comment:
Doesn't zero mean no timeout?

Reply:
I was under the impression it still times out at the supervisor timeout, this being just an additional (shorter) timeout. But this function does wait forever, just the internal loop continues after the supervisor timeout.